include/zephyr/bluetooth/audio/audio.h

```diff
@@ -1354,6 +1425,116 @@ struct bt_audio_unicast_client_cb {
 	void (*available_contexts)(struct bt_conn *conn,
 				   enum bt_audio_context snk_ctx,
 				   enum bt_audio_context src_ctx);
+
+	/**
+	 * @brief Callback function for bt_audio_stream_config() and bt_audio_stream_reconfig().
+	 *
+	 * Called when the codec configure operation is completed on the server.
+	 *
+	 * @param stream   Stream the operation was performed on.
+	 * @param rsp_code Response code.
+	 * @param reason   Reason code.
+	 */
+	void (*config)(struct bt_audio_stream *stream,
+		       enum bt_ascs_rsp_code rsp_code,
+		       enum bt_ascs_reason reason);
```

Comment:
Did you consider adding dedicated container, like:
```
struct bt_ascs_rsp {
	 enum bt_ascs_rsp_code code;
	 enum bt_ascs_reason reason;
};
```

Reply:
I haven't :) What advantages do you see of that approach?